src/filesystem/impls/filer/UrlCache.js

```diff
@@ -61,11 +61,20 @@ define(function (require, exports, module) {
     /**
      * BlobUrlProvider uses Blob URLs in browsers that don't support CacheStorage
      */
-    function BlobUrlProvider() {
+    function BlobUrlProvider(warn) {
         UrlProviderBase.call(this);
 
         this.baseUrl = window.location.href;
         this.shouldRewriteUrls = true;
+
+        // Warn that some network features won't work fully
+        if(warn) {
+            console.info("[Bramble] Your browser doesn't support Service Workers and/or CacheStorage. "  +
```

Comment:
Either is [technically fine](https://dxr.mozilla.org/mozilla-central/source/dom/webidl/Console.webidl#21), but I think you're right that `warn` does what I want more clearly.  I'll update.